src/DssSpell.sol

```diff
@@ -46,41 +96,259 @@ contract SpellAction {
+
+        // Set ilks in MCD_IAM_AUTO_LINE
+        DssAutoLineAbstract(MCD_IAM_AUTO_LINE).setIlk(
+            "ETH-B", 50_000_000 * RAD, 5_000_000 * RAD, 12 hours
+        );
+
+        // Add MCD_IAM_AUTO_LINE
+        CHANGELOG.setAddress("MCD_IAM_AUTO_LINE", MCD_IAM_AUTO_LINE);
 
         // Bump version
-        CHANGELOG.setVersion("1.2.0");
+        CHANGELOG.setVersion("1.2.1");
+
+        //
+        // Various polling changes
+        //
+
+        // Allow yearn finance to access the YFI/USD OSM
+        OsmAbstract(PIP_YFI).kiss(YEARN_PROXY);
+
+        // Set the YFI-A debt ceiling
+        //
+        // Existing debt ceiling: 20 million
+        // New debt ceiling: 30 million
+        VatAbstract(MCD_VAT).file("YFI-A", "line", 30 * MILLION * RAD);
+
+        // Set the YFI-A stability fee
+        //
+        // Previous: 4%
+        // New: 10%
+        JugAbstract(MCD_JUG).drip("YFI-A");
+        JugAbstract(MCD_JUG).file("YFI-A", "duty", TEN_PERCENT_RATE);
 
-        // Set new Chief in the Pause
-        DSPauseAbstract(MCD_PAUSE).setAuthority(MCD_ADM);
+        // Set the USDC-A stability fee
+        //
+        // Previous: 4%
+        // New: 0%
+        JugAbstract(MCD_JUG).drip("USDC-A");
+        JugAbstract(MCD_JUG).file("USDC-A", "duty", ZERO_PERCENT_RATE);
 
-        // Set new Chief in the FlipperMom
-        FlipperMomAbstract(FLIPPER_MOM).setAuthority(MCD_ADM);
+        // Set the TUSD-A stability fee
+        //
+        // Previous: 4%
+        // New: 0%
+        JugAbstract(MCD_JUG).drip("TUSD-A");
+        JugAbstract(MCD_JUG).file("TUSD-A", "duty", ZERO_PERCENT_RATE);
 
-        // Set new Chief in the OsmMom
-        OsmMomAbstract(OSM_MOM).setAuthority(MCD_ADM);
+        // Set the PAXUSD-A stability fee
+        //
+        // Previous: 4%
+        // New: 0%
+        JugAbstract(MCD_JUG).drip("PAXUSD-A");
+        JugAbstract(MCD_JUG).file("PAXUSD-A", "duty", ZERO_PERCENT_RATE);
 
-        // Set Pause delay to 48 hours
-        DSPauseAbstract(MCD_PAUSE).setDelay(48 * 60 * 60);
+        // Set the GUSD-A stability fee
+        //
+        // Previous: 4%
+        // New: 0%
+        JugAbstract(MCD_JUG).drip("GUSD-A");
+        JugAbstract(MCD_JUG).file("GUSD-A", "duty", ZERO_PERCENT_RATE);
+
+        // Set the global debt ceiling
+        VatAbstract(MCD_VAT).file("Line", 1_608_750_000 * RAD);
     }
 }
 
 contract DssSpell {
-    ChainlogAbstract constant CHANGELOG = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
-    address MCD_PAUSE = CHANGELOG.getAddress("MCD_PAUSE");
+    ChainlogAbstract constant CHANGELOG =
+        ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
 
-    // SCD contracts that has the old chief as authority
-    address constant SAI_MOM = 0xF2C5369cFFb8Ea6284452b0326e326DbFdCb867C;
-    address constant SAI_TOP = 0x9b0ccf7C8994E19F39b2B4CF708e0A7DF65fA8a3;
-    //
+    address MCD_PAUSE = CHANGELOG.getAddress("MCD_PAUSE");
```

Comment:
Ca we set these as `immutable` and set them in a constructor?

Reply:
We are still on 0.5.12

Reply:
Are we on `0.6.11` yet?  I was testing and compiling with `0.5.12`.  I think we should build the next one on `0.6.11`.

Reply:
Yeah we should upgrade spells to 0.6 soon.

Reply:
ok, next time